docs/monitors/elasticsearch-query.md

```diff
@@ -0,0 +1,282 @@
+<!--- GENERATED BY gomplate from scripts/docs/templates/monitor-page.md.tmpl --->
+
+# elasticsearch-query
+
+Monitor Type: `elasticsearch-query` ([Source](https://github.com/signalfx/signalfx-agent/tree/master/pkg/monitors/elasticsearch/query))
+
+**Accepts Endpoints**: **Yes**
+
+**Multiple Instances Allowed**: Yes
+
+## Overview
+
+This monitor metricizes aggregated responses from Elasticsearch. The monitor
+constructs SignalFx datapoints based on Elasticsearch aggregation types and
+also aggregation names.
+
+An simple configuration looks like the following:
+
+```yaml
+monitors:
+- type: elasticsearch-query
+ host: localhost
+ port: 9200
+ index: <name_of_index>
+ elasticsearchRequest: |
+    {
+      "query" : {
+        "range" : {
+          "@timestamp" : {
+            "gte": "now-5m"
+          }
+        }
+      },
+      "aggs": {
+        "avg_cpu_utilization": {
+          "avg": {
+            "field": "cpu_utilization"
+          }
+        }
+      }
+    }
+```
+
+The `elasticsearchRequest` take in a request in the format specified
```

Comment:
Point out that it takes the request as a **string** . If you remove the `|` from your example config above it will be valid YAML since JSON is a subset of YAML.  Of course, agent config validation would catch it.